gcp/gcp.go

```diff
@@ -718,8 +742,13 @@ func (gcp *GoogleCloudPrint) assembleJob(job *Job) (*cdd.CloudJobTicket, string,
 
 	gcp.downloadSemaphore.Acquire()
 	t := time.Now()
+	var downloadUrl string
+	if downloadUrl = job.FileURL; !strings.HasPrefix(job.FileURL, "http") {
```

Comment:
is there a reason you assign downloadUrl within the 'if' rather than the live above when you declare downloadUrl?

Reply:
Nothing special, just variable declaration followed by assignment.

Fixed.